pkg/api/proto/documents.proto

```diff
@@ -226,6 +226,13 @@ message DocumentProofResponse {
   schema.VerifiableTxV2 verifiableTx = 5;
 }
 
+message DocumentDeleteRequest {
+  string collection = 1;
+  Query query = 2;
```

Comment:
maybe we can add a `limit` , even an `offset`,  the sql delete statement already support it

Reply:
It's confusing, and imo, better to keep it simple for now.

Reply:
This is single document delete, I am assuming we are in sync with that